Use rust tls (#92)
HQSquantumsimulations/qoqo_iqm: CHANGELOG.md

## Not released

## 0.11.1

> * Use Rust tls instead the openssl one's that were causing an error during deployments.

Reviewer: ```suggestion
* Use Rust tls instead of the openssl one, as it was causing an error during deployments.
```
